docs/developer/centrifuge-api/overview/index.md

```diff
@@ -0,0 +1,203 @@
+---
+id: overview
```

Comment:
Since there's only one page, I would remove the Overview subpage and just make this the contents of `centrifuge-api`. See https://2049f790.documentation-569.pages.dev/user/issuer/ for another page that is on its own.